controllers/goharbor/registry/deployments.go

```diff
@@ -559,6 +576,24 @@ func (r *Reconciler) ApplyGcsStorageEnvs(ctx context.Context, registry *goharbor
 	return nil
 }
 
+func (r *Reconciler) ApplyAliOSSStorageEnvs(ctx context.Context, registry *goharborv1.Registry, deploy *appsv1.Deployment) error {
+	regContainer := &deploy.Spec.Template.Spec.Containers[registryContainerIndex]
+
+	regContainer.Env = append(regContainer.Env, corev1.EnvVar{
+		Name: "REGISTRY_STORAGE_OSS_ACCESSKEYSECRET",
```

Comment:
REGISTRY_STORAGE_ALIOSS_ACCESSKEYSECRET  ?

Reply:
I think it should be consistent with helm.
https://github.com/goharbor/harbor-helm/blob/master/templates/registry/registry-secret.yaml#L33